<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollArea" name="drawViewScrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>711</width>
      <height>661</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>709</width>
       <height>659</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="groupsListWidget">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>251</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="componentsListWidget">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>330</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenByMenuBar"/>
    <addaction name="actionSave"/>
    <addaction name="actionExitByMenuBar"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuAdd_Component">
     <property name="title">
      <string>Add Component</string>
     </property>
     <addaction name="actionCube"/>
     <addaction name="actionPyramid"/>
     <addaction name="actionSphere"/>
     <addaction name="actionLine"/>
    </widget>
    <addaction name="menuAdd_Component"/>
    <addaction name="actionAdd_Group"/>
    <addaction name="actionRedo"/>
    <addaction name="actionUndo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenFolderByToolBar"/>
   <addaction name="actionExitByToolBar"/>
  </widget>
  <action name="actionOpenByMenuBar">
   <property name="icon">
    <iconset>
     <normalon>:/toolButton_images/asset images/folder_open_48px.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExitByMenuBar">
   <property name="icon">
    <iconset>
     <normalon>:/toolButton_images/asset images/exit_48px.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpenFolderByToolBar">
   <property name="icon">
    <iconset resource="asset.qrc">
     <normaloff>:/toolButton_images/asset images/folder_open_48px.png</normaloff>:/toolButton_images/asset images/folder_open_48px.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExitByToolBar">
   <property name="icon">
    <iconset>
     <normalon>:/toolButton_images/asset images/exit_48px.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAdd_Group">
   <property name="text">
    <string>Add Group</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionCube">
   <property name="text">
    <string>Cube</string>
   </property>
  </action>
  <action name="actionPyramid">
   <property name="text">
    <string>Pyramid</string>
   </property>
  </action>
  <action name="actionSphere">
   <property name="text">
    <string>Sphere</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="text">
    <string>Line</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="asset.qrc"/>
 </resources>
 <connections/>
</ui>
